src/Facility.Definition/Fsd/FsdParser.cs

```diff
@@ -18,9 +18,7 @@ public sealed class FsdParser
 		/// <exception cref="ServiceDefinitionException">Thrown if the parse fails.</exception>
 		public ServiceInfo ParseDefinition(NamedText source)
 		{
-			ServiceInfo service;
-			IReadOnlyList<ServiceDefinitionError> errors;
-			if (TryParseDefinition(source, out service, out errors))
+			if (TryParseDefinition(source, out var service, out var errors))
```

Comment:
This is the one new C# 7 feature I don't like. It makes it hard to see where variables are declared, and the scoping is not what I expect. I do like being able to use `var` for out parameters, though. I'm sure I'll get used to it eventually.

Reply:
Yeah, I mostly just do it because ReSharper tells me to. Probably not the best reason. :-)